src/pages/organisationUnits/form/GeometryFields.tsx

```diff
@@ -0,0 +1,75 @@
+import i18n from '@dhis2/d2-i18n'
+import { Field, InputField } from '@dhis2/ui'
+import React from 'react'
+import { useField } from 'react-final-form'
+
+export function GeometryFields() {
```

Comment:
I think we should add a type for `Geometry`. And we get it from `DefinitelyTyped`: https://www.npmjs.com/package/@types/geojson - and use the `Geometry` type.